data.sql

```diff
@@ -90,4 +90,43 @@ FROM owners AS o
 WHERE a.name IN ('Angemon', 'Boarmon') AND o.full_name = 'Dean Winchester';
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221065233-a2a7a5da-25a3-41fd-a4dc-a996d845583d.png)

- [ ] It would be nice if you add some Future Features instead of Coming Soon :smile: You can Features such as :arrow_down: to make your readme file descriptive and professional :blush:

```
Add Indexes to columns
Make Views of tables
```

Reply:
Thank you for your feedback and suggestions, I have implemented them.